### PR TITLE
Feat/product page

### PR DESCRIPTION
**Pull request by ajay-mandaviya**
Screenshot from 2022-02-12 16-27-22
Screenshot from 2022-02-12 16-27-39

### CODE REVIEW

**Review by deepak29-git — APPROVED** [review]
Everything looks good to me

File: pages/ProductPage.html [hunk 1]
```diff
@@ -0,0 +1,541 @@
...
+        <div class="sidebar-filters">
+          <p class="text-bold">Sort By</p>
+          <div>
+            <input type="radio" name="" id="" checked /><label for=""
+              >Price-Low to High</label
+            >
+          </div>
+
+          <div>
+            <input type="radio" name="" id="" /><label for=""
```
Comment (shivanigangadharan) [inline pages/ProductPage.html:92]: You need to give value for the name attribute here, as right now both radio buttons can be selected together.

**Review by shivanigangadharan — APPROVED** [review]
Apart from one thing mentioned below, rest of code looks good to merge.

File: pages/ProductPage.html (hunk 1 above)
Comment (ajay-mandaviya) [reply pages/ProductPage.html:92]: i know , but we don't need as of now

**Pull request merged**